### PR TITLE
Twsf fixes

### PR DESCRIPTION
**Pull request by benetherington**
* Fix interfaces between TWSF service integrations and the database/front end code.
* Do further testing of TWSF service integrations and work out some bugs.
* Update TWSF styling.
* Update templating of all pages, where "update" means "actually use."

### CODE REVIEW

File: database/suggestions.js [hunk 1]
```diff
@@ -111,7 +114,8 @@ module.exports.getSuggestionsWithCountedVotes = async (episode = {}) => {
             voteCount,
         };
     });
-    return formattedCountedSuggestions;
+    if (getEpNum) return [episode.epNum, formattedCountedSuggestions];
+    else return formattedCountedSuggestions;
```
Comment (fonji) [inline database/suggestions.js:118]: Seems like you actually need two functions. I understand the separation is not easy though.

File: src/routes/titles.js [hunk 2]
```diff
@@ -26,11 +31,12 @@ module.exports = (fastify, opts, done) => {
         '/api/titles/:epNum',
         {preHandler: adminPreHandler, logLevel: 'warn'},
         async (request, reply) => {
-            const epNum = request.params.epNum;
-            const countedSuggestions = await getSuggestionsWithCountedVotes({
-                epNum,
-            });
-            return reply.send(countedSuggestions);
+            let requestedEpNum = request.params.epNum; // getSuggestionsWCV defaults to current
```
Comment (fonji) [inline src/routes/titles.js:34]: or you could have done it here
```js
let requestedEpNum = request.params.epNum || await getCurrentEpNum();
```
Would have made `getSuggestionsWithCountedVotes` simpler

File: src/views/layout.pug [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+                    form(action="/logout" method="post")
+                        button(type="submit") Log out
+                else
+                    b Guest
+                    form(action="/login")
+                        button(type="submit") Log in
+        div#title-bar
+            block title-bar
+                h2#title.card This Week in Spaceflight History
+                h2#episode-number.card #{epNum||999}
```
Comment (fonji) [inline src/views/layout.pug:31]: why is this the default?

**Review by fonji — COMMENTED** [review]
I just ran very quickly through this. Seems pretty cool!

File: src/views/layout.pug (hunk 3 above)
Comment (benetherington) [reply src/views/layout.pug:31]: I don't think I need a default anymore. This was helpful while I was writing the CSS, but I think it can get pulled out.

**Pull request merged**